bin/ci-checks-js.sh

```diff
@@ -14,8 +14,8 @@ if [[ -z "${CHECK_CORRECTNESS}" ]] && [[ -z "${CHECK_TESTS}" ]] ; then
 fi
 
 if [ "$CHECK_CORRECTNESS" = true ] ; then
-  npm run flow || pFail
-  npm run lint || pFail
+  yarn run flow || pFail
```

Comment:
What do you think about using the `yarn <script name>` form instead of the `yarn run <script name>` one? The former is the one we typically use in the project.

Reply:
Makes sense 👍